src/fs.js

```diff
@@ -1,8 +1,8 @@
 
 const _ = require('underscore-plus');
 const fs = require('fs-plus');
+const fsPromise = require("node:fs/promises");
```

Comment:
I'm usually in favor of the prefixed core modules thing `require(node:fs/promises)` but I think these were added in Node 16 and backported to a late Node 14.18 minor version, per:
https://nodejs.org/api/modules.html#core-modules

And Electron 12 is based on Node 14.16, according to this?: https://www.electronjs.org/docs/latest/tutorial/electron-timelines

To keep the possibility open of porting all ppm code to run on Pulsar core's copy of Electron, I think this usage would be a blocker.

If we update to newer Electron in core, or if this isn't a concern, we can use the prefixed syntax.

```suggestion
const fsPromise = require("fs/promises");
```

Reply:
Great catch there! Forget how new the prefixed module naming is, I'll add your suggestion thanks!

Reply:
It slightly grabbed me in an OCD-kind-of way that it's `const fsPromise` instead of `const fsPromises` with the second `s`, but I can deal with it, lol.

Reply:
I got you DeeDee